### PR TITLE
Automate tengu feats

### PR DESCRIPTION
**Pull request by Shandyan**
Plus a missed orc one, and doing a better implementation of the champion Blade of Justice feat.

For the Tengu Feather Fan, the upgrade feats have a rollable DC button for the spell they give, with the correct type of save and traits.

### CODE REVIEW

File: packs/data/feat-effects.db/effect-soaring-flight.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "_id": "ytG5XJmkOnDOTjNN",
+    "img": "systems/pf2e/icons/spells/angel-form.webp",
+    "name": "Effect: Soaring Flight",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>To be a tengu is to be unburdened by the concerns of the world below. You grow a pair of magical wings or expand your existing ones. For 5 minutes, you gain a @Compendium[pf2e.actionspf2e.Fly]{Fly} Speed equal to your land Speed or 20 feet, whichever is greater.</p>"
```
Comment (TikaelSol) [inline packs/data/feat-effects.db/effect-soaring-flight.json:8]:
```suggestion
            "value": "<p>To be a tengu is to be unburdened by the concerns of the world below. You grow a pair of magical wings or expand your existing ones. For 5 minutes, you gain a fly Speed equal to your land Speed or 20 feet, whichever is greater.</p>"
```

File: packs/data/feats.db/blade-of-justice.json [hunk 2]
```diff
@@ -30,18 +29,12 @@
...
-                "option": "blade-justice",
-                "toggleable": true
-            },
             {
                 "diceNumber": 2,
                 "key": "DamageDice",
                 "predicate": {
                     "all": [
-                        "blade-justice"
+                        "target:trait:evil"
```
Comment (TikaelSol) [inline packs/data/feats.db/blade-of-justice.json:37]: This doesn't apply to all strikes vs evil, the feat is a 2 action ability so we have to keep it as a toggle.

File: packs/data/feats.db/tengu-feather-fan.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
...
...engu feather fan</em> to cast tengu magic. If your fan is lost or destroyed, you can create a replacement during your daily preparations; if you do so, your previous fan falls apart into mundane feathers. Further feats might grant you additional activations of your <em>tengu feather fan</em>, but you can never have more than three activations per day, no matter how many such feats you have. The DC for spells you cast with your <em>tengu feather fan</em> is your class DC or spell DC, whichever is higher. You can cast the 1st-level <em>@Compendium[pf2e.spells-srd.Gust of Wind]{Gust of Wind}</em> spell by activating your <em>tengu feather fan</em>. @Check[type:fortitude|dc:resolve(@actor.attributes.classOrSpellDC.value)|traits:air]{Fortitude saving throw}</p>\n<hr />\n<p><strong>Activate</strong> Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You cast one of the spells contained in your t<em>engu feather fan</em>. Activating the fan takes the spell's normal number of actions. You can also Activate the fan to cast a cantrip you've gained from a heritage or ancestry feat; this activation doesn't count against the fan's uses per day.</p>"
```
Comment (TikaelSol) [inline packs/data/feats.db/tengu-feather-fan.json:16]: Need to add the other traits to the inline, `evocation,item:trait:spell,primal`

Will need to do this for all of them.

File: packs/data/feats.db/blade-of-justice.json (hunk 2 above)
Comment (Shandyan) [reply packs/data/feats.db/blade-of-justice.json:37]: Oops! Thanks for catching that one

**Pull request merged**